src/App.js

```diff
@@ -1,13 +1,23 @@
 import React from "react";
-
+import Heading from "./Heading";
 import Bookings from "./Bookings";
+import TouristInfoCards from "./TouristInfoCards";
+import Footer from "./Footer";
 import "./App.css";
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <TouristInfoCards />
       <Bookings />
+      <Footer
+        Address={[
```

Comment:
I'd maybe call this "addresses" (because it's an array) or even "contact_details" (because they're not all addresses), rather than "Address" :)